sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -16,11 +16,6 @@ export interface GetSchemaOptions extends OperationOptions {
 export interface GetSchemaPropertiesOptions extends OperationOptions {
 }
 
-// @public
-export const enum KnownSchemaFormat {
-    Avro = "avro"
-}
-
```

Comment:
Commenting here, but I thought we agreed on "schemaName" for the name in SchemaDescription.

Reply:
I am not addressing the comments on the public APIs in this PR. I will meet with @bterlson first on Monday and will open another PR with the agreed-on changes with him. This PR is limited to the regeneration with the latest swagger and handling the fallout.